src/plugins/management/public/components/management_app/_management_app.scss

```diff
@@ -0,0 +1,5 @@
+
+// Hack because the management wrapper is flat HTML and needs a class
+.mgtPage__body {
+  max-width: map-get($euiBreakpoints, 'xl');
+}
```

Comment:
Because of where this is loaded, there was a special override for the Transform Wizard that is no longer in the right cascade position. 

<img width="1937" alt="Screen Shot 2020-05-28 at 3 47 16 PM" src="https://user-images.githubusercontent.com/549577/83187110-a09b7700-a0fb-11ea-9878-edd7a8f956f8.png">

Since both classes simply adjust the max-width, I'd recommend allowing the Transform Wizard to **replace** the default class instead of appending a custom one.

Reply:
Now we replace styles.

Reply:
Also gave this a test locally with the latest commit and can confirm that the transform pages again have proper widths and margins. Thanks!